src/main/java/KieTwoForOne.java

```diff
@@ -0,0 +1,171 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class KieTwoForOne {
+
+    private static ArrayList<Task> tasks = new ArrayList<>(100);
+    static String separationLine = "_________________________________________";
+    static String chatBotName = "KieTwoForOne";
+
+    public enum Instructions {
+        LIST, MARK, UNMARK, BYE, TODO, EVENT, DEADLINE, DELETE
+    }
+
+    public static void addTasks(Task newTask) {
+        tasks.add(newTask);
+        System.out.println("Got it. I've added this task:");
+        System.out.println("    " + newTask);
+        System.out.println(String.format("Now you have %d tasks in the list.", tasks.size()));
+        System.out.println(separationLine);
+    }
+
+    public static void deleteTask(int position) {
+        Task removedTask = tasks.get(position - 1);
+        System.out.println("Noted. I've removed the task");
+        System.out.println("    " + removedTask);
+        tasks.remove(position - 1);
+        System.out.println(String.format("Now you have %d tasks in the list.", tasks.size()));
+        System.out.println(separationLine);
+    }
+    public static void listTasks() {
+        System.out.println("Here are the tasks in your list:");
+        for (int i = 0; i < tasks.size(); i++) {
+            System.out.println(String.format("%d. %s", i + 1, tasks.get(i).toString()));
+        }
+        System.out.println(separationLine);
+    }
+
+    public static void markTask(int position) {
+        Task markedTask = tasks.get(position - 1);
+        System.out.println("Nice! I've marked this task as done:");
+        System.out.println("    " + markedTask.markTask());
+        System.out.println(separationLine);
+    }
+
+    public static void unmarkTask(int position) {
+        Task unmarkedTask = tasks.get(position - 1);
+        System.out.println("Ok. I've marked this task as not done yet:");
+        System.out.println("    " + unmarkedTask.unmarkTask());
+        System.out.println(separationLine);
+    }
+
+    public static boolean isCompleteInput(String[] input) throws KieTwoForOneException {
+        if (input.length <= 1 && !input[0].equalsIgnoreCase("list") && !input[0]. equalsIgnoreCase("bye")) {
+            throw new KieTwoForOneException();
+        }
+        return true;
+    }
+
+    public static boolean isCompleteEventInput(String[] input) throws KieTwoForOneException {
+        if (input.length != 3) {
+            throw new KieTwoForOneException();
+        }
+        return true;
+    }
+
+    public static boolean isCompleteDeadlineInput(String[] input) throws KieTwoForOneException {
+        if (input.length != 2) {
+            throw new KieTwoForOneException();
+        }
```

Comment:
Perhaps could change to 
isInputComplete, isEventInputComplete, and isDeadlineInputComplete
to sound more explicit